common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -179,4 +187,10 @@ object Messages {
   implicit val deviceStatusMessageLike = MessageLike[DeviceUpdateStatus](_.device.show)
 
   implicit val campaignLaunchedMessageLike = MessageLike[CampaignLaunched](_.updateId.show)
+
+  implicit val deltaGenerationRequestMessageLike = MessageLike[DeltaRequest](_.id.show)
+
+  implicit val deltaGeneratedMessageLike = MessageLike[GeneratedDelta](_.namespace.get)
```

Comment:
I wonder if `_.id.show` is a better id for `GeneratedDelta` (This name will be used in the log when an event is being processed)